### PR TITLE
Recognize if a suru verb is followed by suru and merge with the verb stem (#180)

### PR DESCRIPTION
**Pull request by SaltfishAmi**
Untitled

issue #180 

also fixed deinflect for vs-c (suru verbs ending in す instead of する), like 介す

And again, the duplicate line of `Originally based on Rikaichan 1.07` in `data.ts` was not done by me (as far as I can recall), but I wonder why `prettier` would do such a thing.

### CODE REVIEW

File: src/deinflect.ts [hunk 1]
```diff
@@ -153,17 +158,17 @@ const deinflectRuleData: Array<[string, string, number, number]> = [
   ['こうたり', 'こう', 640, DeinflectReason.Tari],
   ['こさせる', 'くる', 2049, DeinflectReason.Causative],
   ['こられる', 'くる', 2049, DeinflectReason.PotentialOrPassive],
-  ['しすぎる', 'す', 513, DeinflectReason.Sugiru],
+  ['しすぎる', 'す', 4609, DeinflectReason.Sugiru],
   ['しすぎる', 'する', 4097, DeinflectReason.Sugiru],
-  ['しちゃう', 'す', 514, DeinflectReason.Chau],
+  ['しちゃう', 'す', 4609, DeinflectReason.Chau],
```
Comment (birtles) [inline src/deinflect.ts:163]: Should this be 4610?

i.e. we used to have:
to: WordType.Godan
from: WordType.Godan
= 00000010 00000010
= 514

But now we have:
to: WordType.Godan or WordType.SuruVerb
from: WordType.Godan
= 00010010 00000010
= 4610

**Comment by birtles** [comment]
> And again, the duplicate line of `Originally based on Rikaichan 1.07` in `data.ts` was not done by me (as far as I can recall), but I wonder why `prettier` would do such a thing.

That's really odd. I've never seen prettier misbehave like that. Do you mind trying to re-install prettier (e.g. `rm -rf node_modules && yarn install`)?

**Comment by birtles** [comment]
This looks great. Ideally it would be best to have separate PRs for the range update and the vs-c handling since I don't think they're necessarily interdependent?

Also, could we add a test for deinflecting vs-c that fails without these changes? So far I'm having trouble finding a case where the result differs, perhaps because JMdict has entries for both 介す and 介する.

**Comment by SaltfishAmi** [comment]
I added the test and splitted them up to #450 . Now we have some spaghetti branch trees.

**Pull request merged**